common/worker.go

```diff
@@ -69,9 +69,9 @@ func (w *Worker) Evaluate(pageFunc goja.Value, args ...goja.Value) any {
 }
 
 // EvaluateHandle evaluates a page function in the context of the web worker and returns a JS handle.
-func (w *Worker) EvaluateHandle(pageFunc goja.Value, args ...goja.Value) api.JSHandle {
+func (w *Worker) EvaluateHandle(pageFunc goja.Value, args ...goja.Value) (api.JSHandle, error) {
 	// TODO: implement
-	return nil
+	return nil, nil
```

Comment:
Could we use `ErrFatal` instead of `nil` here?

Reply:
Makes sense 👍

Reply:
Fatalized here: https://github.com/grafana/xk6-browser/pull/826/commits/04b2e667d203a5b95cf097fc087b480548dcd0e2